iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/DataBrokerProtectionIOSManager.swift

```diff
@@ -226,6 +241,16 @@ public final class DataBrokerProtectionIOSManager {
             }
             queueManager.startScheduledAllOperationsIfPermitted(showWebView: false, jobDependencies: jobDependencies, errorHandler: nil) {
                 Logger.dataBrokerProtection.log("All operations completed in background task")
+                let timeTaken = Date.now.timeIntervalSince(startTime)
+                Logger.dataBrokerProtection.log("Background task finshed all operations with time taken: \(timeTaken)")
+// This should never ever go to production due to the deviceID and only exists for internal testing
+#if DEBUG || ALPHA
+                self.iOSPixelsHandler.fire(.backgroundTaskEndedHavingCompletedAllOperations(
+                    duration: timeTaken * 1000.0,
+                    deviceID: DataBrokerProtectionSettings.deviceIdentifier))
+#endif
+
+                self.scheduleBGProcessingTask()
```

Comment:
I also noticed a we don't schedule a new background task if we complete all jobs before the expiration handler fires